src/utils/node/pm2.ts

```diff
@@ -0,0 +1,129 @@
+import * as path from 'path';
+import { connect, describe, disconnect, list, ProcessDescription, restart, start, stop } from 'pm2';
```

Comment:
what's your take on promisifying the whole `pm2` library using this http://bluebirdjs.com/docs/api/promisification.html
that way we dont have do the below promisification.

Reply:
prefer not to use bluebird, and i tried `util` promisify, but it seems like typescript didnt like that...

Reply:
yeah I tried `util` promisify it's bit ugly, ok if its ok this way then fine.